src/lib.rs

```diff
@@ -315,6 +315,14 @@ impl<T> Sender<T> {
         }
     }
 
+
+    /// Returns true if the associated Receiver handle has been dropped.
+    /// If true is returned, a future call to send is guaranteed to return an error.
+    pub fn is_closed(&self) -> bool {
+        let channel = unsafe { self.channel_ptr.as_ref() };
+        channel.state.load(Acquire) == DISCONNECTED
```

Comment:
I believe the ordering can be `Relaxed`. We don't need to synchronize with any data outside the atomic state variable, we just need the state itself